README.Rmd

```diff
@@ -80,12 +80,6 @@ Take a look at some resources:
 
 We are looking for people to help write tests and vignettes!  You can also take a look at the open [issues](https://github.com/rladies/meetupr/issues).
 
-In order to run our tests, you will have to set the `urlname` for meetup you belong to as an environment variable using the following code:
```

Comment:
This doesn't seem to be true at the moment / anymore?

Reply:
I don't recall what this was used for and it seems dated, so I am just going to say "yes, its not true".